openquakeplatform_taxtweb/templates/taxtweb/incl_taxtweb_inlinecss.html

```diff
@@ -194,6 +194,22 @@
     position: absolute;
 }
 
+#RoofCB4 {
+    width: 85%;
+}
+
+#RoofCB5 {
+    width: 85%;
+}
+
+#FloorCB2 {
+    width: 85%;
+}
+
+#FloorCB3 {
+    width: 85%;
+}
+
```

Comment:
if you want, you can write:

#RoofCB4, #RoofCB5, #FloorCB2, #FloorCB3 {
    width: 85%;
}


But in any case LGTM

Reply:
I will group them in 2 groups Roof-related and Floor-related.